Restructing Changes + Bugfixes

ClarkLach opened the pull request:
Replaced placeholder name "Nick" on main app page to correctly display user's first name. Also updated README to correct dev instructions (though still need to include Firebase info on .env.local somehow)

ClarkLach committed:
Fixed Profile Name and updated README

npease18 commented:
Good catch. I also eliminated the need for .env.local in my auth refactoring PR.

ClarkLach committed:
Links for room markers

Room Marker links to room, deleted unneeded files, beginning work on better structuring

ClarkLach committed:
CSS Fixes

Got rid of all the errors of duplicate colors/sizing.

ClarkLach committed:
Fixed Broken Profile Page

Accidentally broke profile page (since I'm moving the map functions around)

ClarkLach committed:
More Restructure, Formatting all file contents

Breaking stuff into components piece by piece. Also used Prettier on all files in one commit. (Should stick to a consistent, readable format !!)

ClarkLach committed:
Profile Edit into Component

Continuing to break up functionality into separate files

ClarkLach committed:
Commenting

A lil more descriptive, changed a function name as well
